### PR TITLE
Write log using t.Log in registry unit tests

### PR DESCRIPTION
**Pull request by dmage**
cc @legionus @miminar

### CODE REVIEW

File: pkg/dockerregistry/server/repositorymiddleware_test.go [hunk 1]
```diff
@@ -9,8 +9,6 @@ import (
 	"testing"
```
Comment (miminar) [inline pkg/dockerregistry/server/repositorymiddleware_test.go]: Please rename to `repository_test.go`.

**Comment by dmage** [comment]
> Does it pass with the race flag?

Yes, t.Logf is thread-safe.

File: pkg/dockerregistry/server/repositorymiddleware_test.go (hunk 1 above)
Comment (dmage) [reply pkg/dockerregistry/server/repositorymiddleware_test.go]: We already have `repository_test.go`.

Comment (miminar) [reply pkg/dockerregistry/server/repositorymiddleware_test.go]: The `repository.go` [has been recently renamed](https://github.com/openshift/origin/commit/dfbbed95f6beed6173cc657742bdc55c77c34ab1#diff-df06600c3d2e39d32d9dcd471163e52e) from `repositorymiddlware.go`. As a consequence, `repositorymiddleware_test.go` is a leftover now lacking corresponding source file (and my editor's command `:A<CR>` to switch between test and non-test file does not work :cry:).

Can both `repository{middleware,}_test.go` be merged together?

Comment (dmage) [reply pkg/dockerregistry/server/repositorymiddleware_test.go]: Well... Everybody would hate me: `repository_test.go` -> `registry_test.go`, `repositorymiddleware_test.go` -> `repository_test.go`.

Comment (miminar) [reply pkg/dockerregistry/server/repositorymiddleware_test.go]: Not me :-) @legionus would you? Anybody else raise your hand!

Comment (legionus) [reply pkg/dockerregistry/server/repositorymiddleware_test.go]: @dmage @miminar Nop. I'm fine ))

**Comment by dmage** [comment]
flake #16816
/retest

**Pull request merged**